tests/e2e/performance/csi_tests/test_pvc_clone_performance.py

```diff
@@ -551,10 +555,10 @@ def test_pvc_clone_performance_results(self):
             test_count=8,
             test_name="PVC Clone",
         )
-        self.add_test_to_results_check(
-            test="test_pvc_clone_performance_multiple_files",
-            # TODO: after BZ-2101874 will fix, change the test_count to 2
-            test_count=1,
-            test_name="PVC Clone Multiple Files",
-        )
+        # TODO: after BZ-2101874 is fixed, add this code
```

Comment:
The bug is closed, pls see https://bugzilla.redhat.com/show_bug.cgi?id=2101874#c9
Should we introduce changes in the test? i.e., reduce the # of files?

Reply:
@keesturam at some point reducing # of files will of course help. Also increasing the timeout for clone creation with many files. 
I'll have to explore it to find out what number of files works and creation of clone is not stuck. 

Meanwhile this test case is failing each time with the existing number of files. 
Maybe we skip it for now and when I find what is the number that will work - we'll upskip it again ?